pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -12706,6 +12706,18 @@ final: prev:
     meta.homepage = "https://github.com/gcmt/wildfire.vim/";
   };
 
+  windows-nvim = buildVimPluginFrom2Nix {
```

Comment:
this file is generated, please add your plugin to file ./vim-plugin-names

Reply:
Done